ci/sign-all-tasks.yml

```diff
@@ -0,0 +1,45 @@
+parameters:
+  layoutRoot: ''
+  taskFolders: ''
+
+steps:
+# Guard against missing parameters
+- ${{ if eq(parameters.layoutRoot, '') }}:
+  - |
+      Badly configured build; an expected parameter is missing. //
+      layoutRoot: ${{ parameters.layoutRoot }}
+
+# loop over all tasks and create nupkgs
+- script: node ./ci/prepare-tasks-for-signing.js
```

Comment:
We could change this to just run an npm command instead.

Reply:
I assume that would be related to putting it in make.js.